lib/manager/components/version/VersionDateLabel.js

```diff
@@ -4,20 +4,20 @@ import React, {Component} from 'react'
 import {Label as BsLabel} from 'react-bootstrap'
 import moment from 'moment'
 
-import type {FeedVersion} from '../../../types'
+import type {FeedVersionSummary} from '../../../types'
 
 type Props = {
-  version: FeedVersion
+  version: FeedVersionSummary
 }
 
 export default class VersionDateLabel extends Component<Props> {
   render () {
     const {version} = this.props
-    const {validationSummary: summary} = version
+    const {validationResult: summary} = version
     if (!summary) return null
     const now = +moment().startOf('day')
-    const start = +moment(summary.startDate)
-    const end = +moment(summary.endDate)
+    const start = +moment(summary.firstCalendarDate)
+    const end = +moment(summary.lastCalendarDate)
     const future = start > now
```

Comment:
Did we upgrade moment?

Reply:
Could that be handled in #750?

Reply:
Sadly that just got merged. I was more curious why these fields changed, but it's not that important!

Reply:
Actually with https://github.com/ibi-group/datatools-server/commit/a6a72a7890847b13b4d7b604b09b04427e62c974 I reverted changes to this component.